README.md

```diff
@@ -14,6 +14,12 @@ All tests
 
 `npm run test-all`
 
+## Device Configurations
+
+- iPhone 11 Pro `ios.sim.debug-11pro`
+- iPhone 6 `ios.sim.debug-6`
```

Comment:
[nit] Consider more explicit naming (as "6" could be anything). For example "ios.sim.debug.iphone-11-pro", "ios.sim.debug.iphone-6", "ios.sim.debug.iphone-se-2".

Reply:
I was thinking `ios` implied `iphone` hence i didn't add that. Thoughts??

Reply:
Ah, I suppose. I was interpreting that was "ios.sim" - iOS Simulator, "debug" - Debug build, and everything else was the device. Up to you, but "6" could be interpreted as a version or a device type.

Reply:
makes sense - have updated in the next PR. Thanks